Lombiq.Analyzers/general.ruleset

```diff
@@ -386,7 +386,6 @@
     <Rule Id="S3906" Action="Warning" />
     <Rule Id="S3908" Action="Warning" />
     <Rule Id="S3937" Action="Warning" />
-    <Rule Id="S3949" Action="Warning" />
```

Comment:
Why were these removed?

Reply:
These were removed by the VS editor automatically. This happens when rules are removed, or their defaults change. However, I just checked and this is not the case for these rules, so I'll re-add them.